python/bifrost/addon/leda/blocks.py

```diff
@@ -28,6 +28,11 @@
 This file contains blocks specific to LEDA-OVRO.
 """
 
+from __future__ import print_function
+import sys
+if sys.version_info > (3,):
```

Comment:
Might be slightly cleaner to use `six` -- https://six.readthedocs.io/#module-six.moves

Reply:
Yeah, I was trying to avoid adding a dependency.  At some point this should all go away as well since Python2 will be forgotten.

Reply:
Fair enough! This time last year I thought Py2.7 would end up like FITS and still be used in 50 years, but looks more like Py3 is finally prevailing

Reply:
Ha ha, yeah, I was happy to stick with Py2 until major packages started dropping support.

Looking forward I should probably switch the logic of the "if" statements, i.e., `if sys.version_info <= (3,):` to make it easier to strip out later.